docs/api-reference/instances/attributeOut.md

```diff
@@ -0,0 +1,54 @@
+<nav class="fusiondoc-api-breadcrumbs">
+	<a href="../..">Fusion</a>
+	<a href="..">Instances</a>
+</nav>
+
+<h1 class="fusiondoc-api-header" markdown>
+	<span class="fusiondoc-api-icon" markdown>:octicons-code-24:</span>
+	<span class="fusiondoc-api-name">AttributeOut</span>
+	<span class="fusiondoc-api-pills">
+		<span class="fusiondoc-api-pill-type">function</span>
+		<span class="fusiondoc-api-pill-since">since v0.3</span>
+	</span>
+</h1>
+
+Given an attribute name, returns a [special key](./specialkey.md) which outputs the value of
+attribute's with that name. It should be used with a [value](../state/value.md) otherwise an error
```

Comment:
The apostrophe on 'attributes' is grammatically incorrect. Also, the last sentence here should be shortened for simplicity:

'It should be used with a value.'

This mirrors the wording for `Out`'s documentation.